PayPal.podspec

```
@@ -14,12 +14,14 @@ Pod::Spec.new do |s|
   s.subspec "CardPayments" do |s|
     s.source_files  = "Sources/CardPayments/**/*.swift"
     s.dependency "PayPal/CorePayments"
+    s.resource_bundle = { "PayPalCard_PrivacyInfo" => "Sources/CardPayments/PrivacyInfo.xcprivacy" }
```

Looks like this one should be `CardPayments`:
```suggestion
    s.resource_bundle = { "CardPayments_PrivacyInfo" => "Sources/CardPayments/PrivacyInfo.xcprivacy" }
```

---

Oh great catch about the CorePayments and Cards and this error. I just prepended PayPal as opposed to Braintree for all the names. But I agree it's confusing

---

Ah - yeah those are the actual module names on the BT side: https://github.com/braintree/braintree_ios/tree/main/Sources where PPCP doesn't use the prefix in the module names: https://github.com/paypal/paypal-ios/tree/main/Sources. I think we can follow a similar pattern where we use the module names from sources on both.